development.json

```diff
@@ -128,53 +125,6 @@
         "subsidizedRemotes": [
           "arbitrumrinkeby",
           "evmostestnet",
-          "kovan",
-          "neontestnet",
-          "optimismkovan",
-          "rinkeby"
```

Comment:
This makes the diff look weird fyi, but it is correct.  I only removed "kovan" here